### DISCUSSION TITLE
SO import mode as docker installation

### DISCUSSION THREAD

**Discussion by hasamba** [Security-Onion-Solutions/securityonion#13352] [Unsupported Versions] [answered]
i made a project that collaborate several open source projects together, each as docker container, all on one ubuntu server machine.
i would like to add security onion import mode, to add the ability to ingest network log files and analyze them.
i know that the recommended installation comes as ISO but maybe there is a way to install SO as docker that i could not find?
thanks

**Comment by dougburks** [comment, marked as answer]
Security Onion is itself a collection of docker images:
https://docs.securityonion.net/en/2.4/docker.html

Installation via our Security Onion ISO image is our only **supported** installation method. 

You may be able to accomplish your stated goal, but we can't provide any support for it.
